container/container_contract.go

```diff
@@ -391,17 +419,24 @@ func Owner(containerID []byte) []byte {
 func Count() int {
 	count := 0
 	ctx := storage.GetReadOnlyContext()
-	it := storage.Find(ctx, []byte{}, storage.KeysOnly)
+	it := storage.Find(ctx, []byte(containerKeyPrefix), storage.KeysOnly)
 	for iterator.Next(it) {
-		key := iterator.Value(it).([]byte)
-		// V2 format
-		if len(key) == containerIDSize {
-			count++
-		}
+		count++
 	}
 	return count
 }
 
+// Iterate iterates over all container IDs owned by the specified owner.
+func Iterate(owner []byte) iterator.Iterator {
```

Comment:
`ContainersOf`? Frankly, I'm thinking about some NEP-11 compatibility for this contract.

Reply:
Fixed.